### PR TITLE
fix cert-manager sync with ckcp

### PR DESCRIPTION
**Pull request by ramessesii2**
`ckcp`'s deployments depend on `cert-manager` operator. `Cert-manager` is only deployed with the deployment of `Pipeline-Service` on the compute. And thus `ckcp` is dependent upon another service to be deployed on the host cluster.

####  This PR fixes that limitation.

### CODE REVIEW

**Comment by xinredhat** [comment]
LGTM

File: ckcp/openshift_dev_setup.sh [hunk 1]
```diff
@@ -191,22 +192,31 @@ install_openshift_gitops() {
...
   # Certificate resource in order to verify that CRDs are installed and all the
   # required webhooks are reachable by the K8S API server.
-  echo -n "  - openshift-cert-manager-operator: "
+  local i=0
   until kubectl create -f "$CKCP_DIR/openshift/base/certs.yaml" --dry-run=client >/dev/null 2>&1; do
     echo -n "."
     sleep 5
+    i=$((i+1))
+    if [ "$i" == 12 ]; then
+      echo "[ERROR] Failed to deploy Openshift-Cert-Manger operator on the host cluster."
```
Comment (bnallapeta) [inline ckcp/openshift_dev_setup.sh:212]: Openshift-Cert-Manger -> OpenShift cert-manager

**Review by bnallapeta — COMMENTED** [review]
looks good with that minor typo comment. Will let @Roming22 take a look at the PR and merge if all okay.

File: ckcp/openshift_dev_setup.sh (hunk 1 above)
Comment (ramessesii2) [reply ckcp/openshift_dev_setup.sh]: Done, with slight change `OpenShift cert-manager` -->`OpenShift Cert-Manager`

**Review by Roming22 — APPROVED** [review]

**Pull request merged**